src/main/rules/GCI777/javascript/GCI777.asciidoc

```diff
@@ -0,0 +1,21 @@
+== Why is this an issue?
+ 
+Avoid imports of known resource-hungry or deprecated libraries and suggest lighter alternatives.
+
+This rule flags any import, require or dynamic import of libraries listed in the project's deprecation dictionary (`moment`, `iconsax`, etc.), because these libraries
```

Comment:
If this rule depends on specifics libraries, the rule should list the dependencies that are concerned or add an external link to them